.github/workflows/ci.yaml

```
@@ -201,6 +201,35 @@ jobs:
       # launching the more extensive tests below.
       - run: ./build/${{ matrix.target }}/release/installation/${{ matrix.exe }} --help
 
+  write-install-commands:
+    needs: build-standalone
+    runs-on: ubuntu-latest
+    steps:
+      - name: Write installation commands to job summary
+        run: |
+          cat >"$GITHUB_STEP_SUMMARY" <<'~~'
+          Build complete. Commands to install locally:
+
+          Linux:
+
+          ```bash
+          curl -fsSL --proto '=https' https://nextstrain.org/cli/installer/linux | bash -s ci-build/${{ github.run_id }}
+          ```
+
+          macOS:
+
+          ```bash
+          curl -fsSL --proto '=https' https://nextstrain.org/cli/installer/mac | bash -s ci-build/${{ github.run_id }}
+          ```
+
+          Windows:
+
+          ```powershell
+          $script = ((new-object net.webclient).DownloadString('https://nextstrain.org/cli/installer/windows'))
+          Invoke-Command -ScriptBlock ([scriptblock]::Create($script)) -ArgumentList "ci-build/${{ github.run_id }}"
```

Suggest making this more like our recommended installation snippet:

```suggestion
          $env:VERSION = "ci-build/${{ github.run_id }}"
          Invoke-RestMethod https://nextstrain.org/cli/installer/windows | Invoke-Expression
```

or maybe even better, to avoid setting `$env:VERSION`, then something similar to your approach with a script block but more succinct:

```suggestion
          Invoke-Expression "& { $(Invoke-RestMethod https://nextstrain.org/cli/installer/windows) } ci-build/${{ github.run_id }}"
```

(Relatedly, note that `Invoke-RestMethod https://nextstrain.org/cli/installer/windows` could be replaced by `(Invoke-WebRequest https://nextstrain.org/cli/installer/windows).Content` if the "RestMethod" moniker is too displeasing. :upside_down_face:)

---

I used `Invoke-Command` after failing to figure out out `Invoke-Expression` works. Also didn't realize that the `VERSION` env var is supported.

I'll test your approaches and go with the one-liner if it works, to avoid the warning that some terminal apps give on multi-line paste.

---

`VERSION` env var is supported by both installers, but it's nice in interactive examples to have a single line, not clutter the env, and pass an arg directly.

I tested both in `pwsh` on Linux, so the syntax _should_ work. :-)

---

Tested on Windows, works great!